components/ConfirmationModal.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledButton from './StyledButton';
+import Container from './Container';
+import { P } from './Text';
+import Modal, { ModalBody, ModalHeader, ModalFooter } from './StyledModal';
+
+const ConfirmationModal = ({ show, header, body, onClose, actions }) => (
+  <Modal width="570px" show={show} onClose={() => onClose()}>
```

Comment:
Let's not use a fixed `width`, that doesn't work on mobile.

Reply:
We actually have it that way across our modals.

Reply:
Ah yes I forgot we added a `maxWidth` on the modal, all good then!